frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyMfaCodeHandler.java

```diff
@@ -292,6 +294,9 @@ private APIGatewayProxyResponseEvent verifyCode(
                                             .setVerifiedMfaMethodType(
                                                     codeRequest.getMfaMethodType()));
 
+                            authSessionService.setVerifiedMfaMethodType(
+                                    authSessionId, codeRequest.getMfaMethodType());
+
```

Comment:
Same migration related question as previous. Is something in hand already for when the line above this one will be removed?